src/borg/archiver.py

```diff
@@ -1363,10 +1363,11 @@ def _list_repository(self, args, manifest, write):
         output_data = []
 
         for archive_info in manifest.archives.list_considering(args):
+            archive = Archive(repository, key, manifest, archive_info.name)
```

Comment:
(Depending on how large the archives are: A couple dozen kB times a few hundred to thousand = a couple dozen MB download for *each* borg list, independent of the RTT issue).